src/qtt/algorithms/fitting.py

```diff
@@ -248,10 +249,11 @@ def gaussian_model(x, mean, sigma, amplitude):  # type: ignore
             y = gaussian(x, mean, sigma, amplitude)
             return y
 
+    lmfit_method = 'least_squares'
```

Comment:
Check:  Shouldn't fit method in fit_double_gaussian also change, because fit_gaussian and fit_double_gaussian are both used in refit_double_gaussian, each using a different minimizer now.

Reply:
I don't believe that is a problem, but I changed it anyway.